external/js/cothority/src/calypso/calypso-rpc.ts

```diff
@@ -105,6 +105,13 @@ export class LongTermSecret extends OnChainSecretRPC {
             roster = bc.getConfig().roster;
         }
         const ocs = new OnChainSecretRPC(bc);
+        try {
+            await ocs.authorizeRoster(roster);
+        } catch (e) {
+            if (e.toString().indexOf("already authorised") < 0) {
```

Comment:
If this is the only way of checking it, I would at least comment the code where the error is raised to indicate this dangerous coupling.

Reply:
This is raised in `dedis/cothority/byzcoin`. Any better idea how the coupling can be removed?

Reply:
I don't have a better idea, I would simply add a comment where you mentioned, and update the test to capture this new behaviour.

Reply:
Done.